src/libraries/System.Collections.Concurrent/tests/BlockingCollectionTests.cs

```diff
@@ -966,6 +966,52 @@ public static void Test21_CopyToExceptions()
             });
         }
 
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
+        public static void Test_MaxSize(bool sta)
+        {
+            Thread t = new Thread(() =>
+            {
+                BlockingCollection<int>[] blockingCollections = new BlockingCollection<int>[63];
+                for (int i = 0; i < blockingCollections.Length; ++i)
+                {
+                    blockingCollections[i] = ConstructBlockingCollection<int>(1);
+                    blockingCollections[i].Add(1);
+                }
+                Assert.Equal(-1, BlockingCollection<int>.TryAddToAny(blockingCollections, 1, 1));
```

Comment:
Can you also parameterize the test to pass in a cancelable cancellation token as well? e.g. the result of `new CancellationTokenSource().Token`.  Same for the test below.

Reply:
Actually, nevermind, with the current implementation there's always a token used regardless of whether one is provided externally, so with the current implementation, it won't matter.